resources/schema.py

```diff
@@ -862,7 +888,9 @@ class Mutation:
         description="The `imageFile` field takes an image as input. "
         "To provide the file, you have to perform the request with a client "
         "that conforms to the [GraphQL Multipart Request Spec]"
-        "(https://github.com/jaydenseric/graphql-multipart-request-spec)."
+        "(https://github.com/jaydenseric/graphql-multipart-request-spec).\n\n"
+        "Errors:"
+        "\n* removeHarborMap is passed"
```

Comment:
we probably do not raise this error anymore?

Reply:
I knew I was missing something in the docs. 🤔